Library/Homebrew/test/utils/github_spec.rb

```diff
@@ -6,12 +6,11 @@
 describe GitHub do
   describe "::search_code", :needs_network do
     it "queries GitHub code with the passed parameters" do
-      results = described_class.search_code(repo: "Homebrew/brew", path: "/",
-                                            filename: "readme", language: "markdown")
+      results = described_class.search_code(user: "Homebrew", path: "Casks", filename: "1password-cli")
 
       expect(results.count).to eq(1)
-      expect(results.first["name"]).to eq("README.md")
-      expect(results.first["path"]).to eq("README.md")
+      expect(results.first["name"]).to eq("1password-cli.rb")
+      expect(results.first["path"]).to eq("Casks/1password-cli.rb")
```

Comment:
Please leave this as-is; I'd like to see this PR not require any test changes.

Reply:
Sorry, I'll reset this test.

Reply:
No problem!

Reply:
@hyuraku Any news here?

Reply:
@MikeMcQuaid 
I just updated!